src/rules/function-url-quotes/__tests__/index.js

```diff
@@ -72,146 +76,7 @@ testRule(rule, {
     code: "a { background-image: url(\"foo.css\"), url(\"bar.css\"), url(\"baz.css\"); }",
   }, {
     code: "a { background-image: url( \"foo.css\" ), url( \"bar.css\" ), url( \"baz.css\" ); }",
```

Comment:
Ok, makes sense. It just wasn't obvious in the diffs that those tests were still there.